.github/workflows/check-dist-sha256.yml

```diff
@@ -0,0 +1,51 @@
+name: Check dist sha256
+# Validates whether the dist folder is as expected.
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
```

Comment:
Please add default permissions. This is a good scorecard pre-submit check we could built (not sure how to capture which checks should bock and which should not though)

Reply:
I have included the permissions. Let's check if things fail.